process/tac_roles.md

```diff
@@ -0,0 +1,38 @@
+---
+parent: Processes
+has_children: true
+---
+
+# TAC Roles
+
+* TOC
+{:toc}
+
+The TAC has several leadership roles, which provide both the TAC and the hosted projects and working groups with 
+support and oversight to ensure a healthy ecosystem. This document outlines those roles and the processes
+regarding appointment and selection.
+
+## Chairperson and Vice Chairperson.
+
+Per the [Academy Software Foundation Directed Fund Charter, Section 7(f)][ASWF Charter], the role of the TAC Chairperson is as follows:
+
+> The TAC representatives will elect a chair to preside over meetings, ensure minutes are 
+taken and drive the TAC agenda with input from the TAC representatives
+
+A Vice Chairperson role has been established to ensure continuity and increase leadership support. The Vice Chairperson both supports and shadows the Chairperson so that they can serve as Chairperson the following year. The chart below illustrates this.
```

Comment:
```suggestion
The Chairperson role serves a term of one year. A Vice Chairperson role has been established to ensure continuity and increase leadership support. The Vice Chairperson both supports and shadows the Chairperson so that they can serve as Chairperson the following year. The chart below illustrates this.
```

Implied by the surrounding language, but better to make it explicit.